src/Public/TraceContext.cs

```diff
@@ -0,0 +1,54 @@
+//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Google.Protobuf.Collections;
+using System;
+using System.Collections;
+using System.Collections.Generic;
+
+namespace Microsoft.Azure.Functions.PowerShellWorker
+{
+    /// <summary>
+    /// Custom TraceContext constructed from the TraceContext members in gRPC.
+    /// </summary>
+    public class TraceContext
+    {
+        /// <summary>
+        /// Constructor for TraceContext.
```

Comment:
Consider removing (auto-generated?) comments like this, as they don't add any value.

Reply:
I was seeing errors if these comments were not added.

Reply:
Oh, I see. One way to suppress these errors is to declare this class `internal`.